### DISCUSSION TITLE
Show Elasticsearch term aggregation in a table

### DISCUSSION THREAD

**Discussion by Jacuo** [grafana/grafana#66119] [Data Sources]
I have aggregation from Elasticsearch metricbeat 
![image](https://user-images.githubusercontent.com/8425800/230377642-947bc241-6dde-4e9c-b50c-ef8e5b229790.png)

But I want to have pod.name as field in row, without select input - so I want to have table with rows for each time / pod.name 
I tried to use some transformations but with no luck 
Is it possible ? How ?

Jacek

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This is phrased as a question, and we'd like questions about grafana to go to our[ community forum](https://community.grafana.com/). Feel free to make an [issue](https://github.com/grafana/grafana/issues) if you cannot find a solution and would like to request the feature.
